flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/InternalSinkWriterMetricGroup.java

```diff
@@ -71,6 +79,16 @@ public Counter getNumRecordsOutErrorsCounter() {
         return numRecordsOutErrors;
     }
 
+    @Override
+    public org.apache.flink.metrics.Counter getNumRecordsWrittenCounter() {
```

Comment:
What's up with the the lack of imports?

Reply:
ops, I have changed the IDEA settings for other issues. I will change it back. Thanks.